gm4_speed_paths/data/gm4_speed_paths/functions/clock/check.mcfunction

```diff
@@ -0,0 +1,10 @@
+# Starts clock if not enabled
+# @s = player, stepping on dirt_path
+# at @s
+# run from advancement, gm4_speed_paths:on_path
+
+execute store result score $running gm4_speed_paths run schedule clear gm4_speed_paths:main
+execute if score $running gm4_speed_paths matches 1 run schedule function gm4_speed_paths:main 8t
```

Comment:
Why does this reschedule the clock when it's already scheduled? If multiple players stepped on a path every 4 ticks staggered for example, could they disable the main clock?

Reply:
Thought I replied to this.  I was trying to figure out a way to both check that the clock is running and not to have this issue by appending the schedule.  Though given the advancement change that uses the tag to block, I could alternatively check the tag for @a, which is the same trigger as scheduling the main function?